sql/core/src/main/scala/org/apache/spark/sql/execution/SparkQl.scala

```diff
@@ -52,26 +57,30 @@ private[sql] class SparkQl(conf: ParserConf = SimpleParserConf()) extends Cataly
           nodeToDescribeFallback(node)
         } else {
           tableType match {
-            case Token("TOK_TABTYPE", Token("TOK_TABNAME", nameParts :: Nil) :: Nil) =>
+            case Token("TOK_TABTYPE", Token("TOK_TABNAME", nameParts) :: Nil) =>
```

Comment:
Why change this? You didn't touch the describe stuff in `SparkSqlParser.g` right?